fragments/master.sh

```diff
@@ -25,6 +25,16 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
     exit 1
 fi
 
+RESTART_POLICY="no"
+START_MODE=$1
+
+if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
```

Comment:
yes, this was my idea ! We also need two different versions of start_k8s(). The rest should be the same